### DISCUSSION TITLE
Island Sanctuary Minion Names

### DISCUSSION THREAD

**Discussion by Ferelily** [Glyceri/FFXIVPetRenamer#11] [Ideas]
Would it be at all possible/plausible to extend the renaming function to the minions you release on your island sanctuary? Any minion I put there reverts back to its default name.

**Comment by Glyceri** [comment]
HMM, I've never played the island sanctuary so I dont really know what this refers to. 
If you can provide me with a little more info such as: 
- What kind of minions, are they different from the normal ones you spawn out in the world? 
- How do I get them?
-  Where would they show up, e.g. they follow you like normal minions, or they stay around at the island sanctuary.
If I know more I will certainly try to add it!

**Reply by Glyceri** [reply]
![image](https://github.com/user-attachments/assets/b9b26ddb-df66-4265-921f-4fdeb3e5b30b)
I found it! This is VERY CUTE and will DEFINITELY be added 😌

**Reply by Glyceri** [reply]
Update: The impelmentation will be slightly janky c:

**Comment by Glyceri** [comment]
![image](https://github.com/user-attachments/assets/526bbd19-c1e3-4dc6-bc7c-709dbf70462b)
c:

**Comment by Ferelily** [comment]
Thank you!!!!
